src/main/scala/tilelink/RegisterRouter.scala

```diff
@@ -81,19 +85,52 @@ case class TLRegisterNode(
     bundleIn.c.ready := Bool(true)
     bundleIn.e.ready := Bool(true)
 
+    genRegDescsJson(mapping: _*)
```

Comment:
I think the convention is no space here `:_*`

Reply:
fixed